config.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 the generic-device-plugin authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Plz sort the imports here. This should really be part of the linting stage of CI

Reply:
Whoops. I usually configure my IDE to use Goimports but I think it didn't enable it for this one. Will fix

Reply:
Hmm actually, goimports leaves the file as is

Reply:
I sorted this one into 3 sections according to your spec below